src/Symfony/Component/DomCrawler/Test/Constraint/CrawlerSelectorTextContains.php

```diff
@@ -45,7 +45,16 @@ protected function matches($crawler): bool
             return false;
         }
 
-        return false !== mb_strpos($crawler->text(null, false), $this->expectedText);
+        $nodeTexts = $crawler->each(function (Crawler $node) {
```

Comment:
I would suggest to remove the call to each here and do everything in the foreach.

Reply:
Agreed, updated